pkg/library/libraryapplyconfiguration/apply_configuration.go

```diff
@@ -66,7 +55,18 @@ func ValidateAllDesiredMutationsGetter(allDesiredMutationsGetter AllDesiredMutat
 	combinedList.GeneratedNameResources = append(combinedList.GeneratedNameResources, allAllowedOutputResources.UserWorkloadResources.GeneratedNameResources...)
 	filteredMutationRequests := FilterSerializedRequests(allMutationRequests, combinedList)
 
-	if unspecifiedOutputResources := manifestclient.DifferenceOfSerializedRequests(allMutationRequests, filteredMutationRequests); len(unspecifiedOutputResources) > 0 {
+	return manifestclient.DifferenceOfSerializedRequests(allMutationRequests, filteredMutationRequests)
+}
+
+func ValidateAllDesiredMutationsGetter(allDesiredMutationsGetter AllDesiredMutationsGetter, allAllowedOutputResources *libraryoutputresources.OutputResources) error {
```

Comment:
maybe this function could also return `unspecifiedOutputResources` ?

Reply:
> maybe this function could also return `unspecifiedOutputResources` ?

I prefer my validators only return errors.